src/components/DragCaptureZone/DragCaptureZone.jsx

```diff
@@ -0,0 +1,148 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+
+const boundClassNames = lucidClassNames.bind('&-DragCaptureZone');
+const {
+	func,
+	string
+} = React.PropTypes;
+
+/**
+ * {"categories": ["utility"]}
+ *
+ * This is a helper component used to capture mouse events to determine when the
+ * user starts, is and stops dragging.
+ */
+const DragCaptureZone = React.createClass({
+	propTypes: {
+		/**
+		 * Appended to the component-specific class names set on the root
+		 * element.
+		 */
+		className: string,
+
+		/**
+		 * Called as the user drags the mouse.
+		 *
+		 * Signature: `({ dx, dy, pageX, pageY }, { event, props }) => {}`
+		 */
+		onDrag: func,
+
+		/**
+		 * Called when the user releases the mouse button after having dragged.
+		 *
+		 * Signature: `({ dx, dy, pageX, pageY }, { event, props }) => {}`
+		 */
+		onDragEnd: func,
+
+		/**
+		 * Called when the user presses the mouse button down while over the
+		 * component.
+		 *
+		 * Signature: `({ dx, dy, pageX, pageY }, { event, props }) => {}`
+		 */
+		onDragStart: func
+	},
+
+	getDefaultProps() {
+		return {
+			onDrag: _.noop,
+			onDragEnd: _.noop,
+			onDragStart: _.noop
+		};
+	},
+
```

Comment:
I don't _think_ that this would happen. While you're dragging around, it's basically a bunch of individual `mousemove` event handlers being fired so if the component were to be unmounted while you were doing that it would just unmount somewhere in between two of those `mousemove` events and you'd lose the response.